django/contrib/admin/checks.py

```diff
@@ -104,7 +104,7 @@ def _check_raw_id_fields_item(self, obj, model, field_name, label):
             return refer_to_missing_field(field=field_name, option=label,
                                           model=model, obj=obj, id='admin.E002')
         else:
-            if not isinstance(field, (models.ForeignKey, models.ManyToManyField)):
+            if not isinstance(field, models.ForeignKey) and not field.many_to_many:
                 return must_be('a ForeignKey or ManyToManyField',
```

Comment:
would we want to adjust the message here to not use `ManyToManyField` as that's technically not completely correct?